packages/starlight/components/SidebarSublist.astro

```diff
@@ -24,13 +24,11 @@ const { sublist, nested } = Astro.props;
 					>
 						<span>{entry.label}</span>
 						{entry.badge && (
-							<>
-								{' '}
-								<Badge
-									text={entry.badge.text}
-									variant={entry.isCurrent ? 'outline' : entry.badge.variant}
-								/>
-							</>
+							<Badge
+								variant={entry.badge.variant}
+								class={entry.badge.class}
+								text={entry.badge.text}
+							/>
```

Comment:
It’s worth double checking that removing the fragment and explicit `{' '}` whitespace here doesn’t cause issues — it was added as a workaround for issues caused by https://github.com/withastro/compiler/issues/852

Reply:
I don’t see the whitespace stripped in the preview, but maybe I’m missing something?

Reply:
Tested a local build in chrome and safari, it seems to be working fine.